src/server/appserver.cpp

```diff
@@ -105,7 +122,8 @@ AppServer::AppServer(int &argc, char **argv)
       _helpAsked(false),
       _versionAsked(false),
       _clearSyncNodesAsked(false),
-      _debugMode(false) {
```

Comment:
I left it as it was previously, but yes, it could be beneficial. Is there a technical advantage to initialize these members it in the header? For better readability?